flow-state/server/rest/endpoints.go

```diff
@@ -60,10 +60,10 @@ func AppendEndpoints(router *httprouter.Router, logger log.Logger, exposeRecorde
 	router.GET("/v1/flows", sm.getFlowNames)
 	router.GET("/v1/apps/:appName/versions", sm.getAppVersions)
 
-	router.GET("/v1/app/state/:app/:version", sm.getAppState)
+	router.GET("/v1/app/state/:app", sm.getAppState)
```

Comment:
@milanbhagwat Lets change path parameter **app** to **appName** for consistency.

Reply:
resolved with commit id : 0eb8923472f739aee95ff6d2bf5441e2cc9fc730